app/src/main/java/org/oppia/android/app/settings/profile/ProfileEditFragmentPresenter.kt

```diff
@@ -115,7 +115,7 @@ class ProfileEditFragmentPresenter @Inject constructor(
     dialogFragment.showNow(fragment.childFragmentManager, TAG_PROFILE_DELETION_DIALOG)
   }
 
-  /** This function is responsible for the deletion of the profile in [ProfileEditFragment]. */
+  /** Requests the specific profile to be deleted from the app and then routes the user either to administrator controls activity or profile list activity depending on whether they are currently using a tablet device. */
```

Comment:
This needs line-wrapping (no lines should exceed 100 characters, including documentation). Please see https://developer.android.com/kotlin/style-guide#formatting_2 for details on KDoc formatting.